index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link href="style.css" rel="stylesheet">
+        <title>Portfolio: Setup and Mobile Version Skeleton</title>
+    </head>
+    <body>
+        <header class="toolbar">
+            <span>Welcome</span>
+            <button type="button" class="btn btn-tertiary"><img src="Images/Normal-Button/Tertiary/Icons/Union.svg" alt="union icon"></button>
+        </header>
+        <section class="headline">
+            <h1>Hey there. I’m<br>Sahar<span class="h1-span">I’m a software developer</span></h1>
```

Comment:
Let's separate these texts into two different tags.
- [ ] Kindly put **`Hey there. I'm Sahar`** in a different `h1` tag.
- [ ] Also put **`I'm a software developer`** in a different `hx` tag